crates/libtiny_tui/src/tui.rs

```diff
@@ -525,6 +498,50 @@ impl TUI {
         self.fix_scroll_after_close();
     }
 
+    pub(crate) fn new_misc_tab(&mut self, misc_tab: MiscTab, idx: Option<usize>) -> Option<usize> {
+        match self.find_misc_tab_idx(misc_tab) {
+            None => {
+                let tab_idx = if let Some(idx) = idx {
+                    // use requested index
+                    idx
+                } else {
+                    // put tab to the right of the current tab or we are the first tab
+                    if self.tabs.is_empty() {
+                        0
+                    } else {
+                        self.active_idx + 1
+                    }
+                };
+                self.new_tab(
+                    tab_idx,
+                    MsgSource::Misc(misc_tab),
+                    true,
+                    if cfg!(feature = "desktop-notifications") {
+                        Notifier::Mentions
+                    } else {
+                        Notifier::Off
+                    },
+                    None,
+                );
+                Some(tab_idx)
+            }
+            Some(idx) => {
+                self.select_tab(idx);
```

Comment:
This is very confusing. Switching to the help tab happens in two entirely different places.

1. Here when tab already exists
2. In `show_help_tab` when tab doesn't already exist

Need to refactor this somehow. I have a suggestion in my top-level comment.

Reply:
I removed the automatic selection fully. Instead, I made the help tab highlight on creation.

Reply:
Why? If I run `/help` I want to see help right now, without any more effort. Why run `/help` if you don't want to see it?

Why not switch to the tab in command implementation?

Reply:
Done.